From a496ed9ac1f925823a7b842e073a3566f0f63b86 Mon Sep 17 00:00:00 2001
From: Nguyen Tran <nguyen.tran.pz@bp.renesas.com>
Date: Wed, 9 Jul 2025 14:31:14 +0700
Subject: [PATCH] pinctrl: renesas: r8a779f0: Clean up unused pin muxes in
 IPxSR registers

Replaced FM(GPX_XX) with F_(0, 0) in IP0SR1, IP0SR4, IP1SR4, and IP2SR4
to match HWUM v1.30 and clearly indicate pins without alternate functions.

Signed-off-by: Nguyen Tran <nguyen.tran.pz@bp.renesas.com>
Reviewed-by: Huy Bui <huy.bui.wm@renesas.com>
---
 drivers/pinctrl/renesas/pfc-r8a779f0.c | 81 +++++++++-----------------
 1 file changed, 27 insertions(+), 54 deletions(-)

diff --git a/drivers/pinctrl/renesas/pfc-r8a779f0.c b/drivers/pinctrl/renesas/pfc-r8a779f0.c
index 2519ca4d6e3..4164e5d0710 100644
--- a/drivers/pinctrl/renesas/pfc-r8a779f0.c
+++ b/drivers/pinctrl/renesas/pfc-r8a779f0.c
@@ -289,38 +289,38 @@
 
 /* IP0SR1 */		/* 0 */			/* 1 */			/* 2 */			/* 3 */			/* 4 */		/* 5 */			/* 6 - F */
 /* TSN2_MDC, TSN2_MDIO  definitions are removed due to SW conflict */
-#define IP0SR1_3_0	FM(GP1_00)		FM(TCLK1)		FM(HSCK2)		F_(0, 0)		F_(0, 0)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP0SR1_7_4	FM(GP1_01)		FM(TCLK4)		FM(HRX2)		F_(0, 0)		F_(0, 0)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP0SR1_11_8	FM(GP1_02)		F_(0, 0)		FM(HTX2)		FM(MSIOF2_SS1)		F_(0, 0)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP0SR1_15_12	FM(GP1_03)		FM(TCLK2)		FM(HCTS2_N)		FM(MSIOF2_SS2)		FM(CTS4_N)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP0SR1_19_16	FM(GP1_04)		FM(TCLK3)		FM(HRTS2_N)		FM(MSIOF2_SYNC)		FM(RTS4_N)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP0SR1_23_20	FM(GP1_05)		FM(MSIOF2_SCK)		FM(SCK4)		F_(0, 0)		F_(0, 0)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP0SR1_27_24	FM(GP1_06)		FM(MSIOF2_RXD)		FM(RX4)			F_(0, 0)		F_(0, 0)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP0SR1_31_28	FM(GP1_07)		FM(MSIOF2_TXD)		FM(TX4)			F_(0, 0)		F_(0, 0)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP0SR1_3_0	F_(0, 0)		FM(TCLK1)		FM(HSCK2)		F_(0, 0)		F_(0, 0)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP0SR1_7_4	F_(0, 0)		FM(TCLK4)		FM(HRX2)		F_(0, 0)		F_(0, 0)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP0SR1_11_8	F_(0, 0)		F_(0, 0)		FM(HTX2)		FM(MSIOF2_SS1)		F_(0, 0)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP0SR1_15_12	F_(0, 0)		FM(TCLK2)		FM(HCTS2_N)		FM(MSIOF2_SS2)		FM(CTS4_N)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP0SR1_19_16	F_(0, 0)		FM(TCLK3)		FM(HRTS2_N)		FM(MSIOF2_SYNC)		FM(RTS4_N)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP0SR1_23_20	F_(0, 0)		FM(MSIOF2_SCK)		FM(SCK4)		F_(0, 0)		F_(0, 0)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP0SR1_27_24	F_(0, 0)		FM(MSIOF2_RXD)		FM(RX4)			F_(0, 0)		F_(0, 0)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP0SR1_31_28	F_(0, 0)		FM(MSIOF2_TXD)		FM(TX4)			F_(0, 0)		F_(0, 0)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
 
 /* IP0SR4 */		/* 0 */			/* 1 */			/* 2 */			/* 3 */			/* 4 */		/* 5 */			/* 6 - F */
-#define IP0SR4_3_0	FM(GP4_00)		FM(MSPI4SC)		F_(0, 0)		FM(TAUD0I2)		FM(TAUD0O2)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP0SR4_7_4	FM(GP4_01)		FM(MSPI4SI)		F_(0, 0)		FM(TAUD0I4)		FM(TAUD0O4)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP0SR4_11_8	FM(GP4_02)		FM(MSPI4SO_MSPI4DCS)	F_(0, 0)		FM(TAUD0I3)		FM(TAUD0O3)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP0SR4_15_12	FM(GP4_03)		FM(MSPI4CSS1)		F_(0, 0)		FM(TAUD0I6)		FM(TAUD0O6)	FM(MSPI5SO_MSPI5DCS)	F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP0SR4_19_16	FM(GP4_04)		FM(MSPI4CSS0)		FM(MSPI4SSI_N)		FM(TAUD0I5)		FM(TAUD0O5)	FM(MSPI5SC)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP0SR4_23_20	FM(GP4_05)		FM(MSPI4CSS3)		F_(0, 0)		FM(TAUD0I8)		FM(TAUD0O8)	FM(MSPI5SSI_N)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP0SR4_27_24	FM(GP4_06)		FM(MSPI4CSS2)		F_(0, 0)		FM(TAUD0I7)		FM(TAUD0O7)	FM(MSPI5SI)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP0SR4_31_28	FM(GP4_07)		FM(MSPI4CSS5)		F_(0, 0)		FM(TAUD0I10)		FM(TAUD0O10)	FM(MSPI5CSS1)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP0SR4_3_0	F_(0, 0)		FM(MSPI4SC)		F_(0, 0)		FM(TAUD0I2)		FM(TAUD0O2)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP0SR4_7_4	F_(0, 0)		FM(MSPI4SI)		F_(0, 0)		FM(TAUD0I4)		FM(TAUD0O4)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP0SR4_11_8	F_(0, 0)		FM(MSPI4SO_MSPI4DCS)	F_(0, 0)		FM(TAUD0I3)		FM(TAUD0O3)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP0SR4_15_12	F_(0, 0)		FM(MSPI4CSS1)		F_(0, 0)		FM(TAUD0I6)		FM(TAUD0O6)	FM(MSPI5SO_MSPI5DCS)	F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP0SR4_19_16	F_(0, 0)		FM(MSPI4CSS0)		FM(MSPI4SSI_N)		FM(TAUD0I5)		FM(TAUD0O5)	FM(MSPI5SC)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP0SR4_23_20	F_(0, 0)		FM(MSPI4CSS3)		F_(0, 0)		FM(TAUD0I8)		FM(TAUD0O8)	FM(MSPI5SSI_N)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP0SR4_27_24	F_(0, 0)		FM(MSPI4CSS2)		F_(0, 0)		FM(TAUD0I7)		FM(TAUD0O7)	FM(MSPI5SI)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP0SR4_31_28	F_(0, 0)		FM(MSPI4CSS5)		F_(0, 0)		FM(TAUD0I10)		FM(TAUD0O10)	FM(MSPI5CSS1)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
 /* IP1SR4 */		/* 0 */			/* 1 */			/* 2 */			/* 3 */			/* 4 */		/* 5 */			/* 6 - F */
-#define IP1SR4_3_0	FM(GP4_08)		FM(MSPI4CSS4)		F_(0, 0)		FM(TAUD0I9)		FM(TAUD0O9)	FM(MSPI5CSS0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP1SR4_7_4	FM(GP4_09)		FM(MSPI4CSS7)		F_(0, 0)		FM(TAUD0I12)		FM(TAUD0O12)	FM(MSPI5CSS3)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP1SR4_11_8	FM(GP4_10)		FM(MSPI4CSS6)		F_(0, 0)		FM(TAUD0I11)		FM(TAUD0O11)	FM(MSPI5CSS2)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP1SR4_15_12	FM(GP4_11)		FM(ERRORIN0_N)		F_(0, 0)		FM(TAUD0I14)		FM(TAUD0O14)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP1SR4_19_16	FM(GP4_12)		FM(ERROROUT_C_N)	F_(0, 0)		FM(TAUD0I13)		FM(TAUD0O13)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP1SR4_3_0	F_(0, 0)		FM(MSPI4CSS4)		F_(0, 0)		FM(TAUD0I9)		FM(TAUD0O9)	FM(MSPI5CSS0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP1SR4_7_4	F_(0, 0)		FM(MSPI4CSS7)		F_(0, 0)		FM(TAUD0I12)		FM(TAUD0O12)	FM(MSPI5CSS3)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP1SR4_11_8	F_(0, 0)		FM(MSPI4CSS6)		F_(0, 0)		FM(TAUD0I11)		FM(TAUD0O11)	FM(MSPI5CSS2)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP1SR4_15_12	F_(0, 0)		FM(ERRORIN0_N)		F_(0, 0)		FM(TAUD0I14)		FM(TAUD0O14)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP1SR4_19_16	F_(0, 0)		FM(ERROROUT_C_N)	F_(0, 0)		FM(TAUD0I13)		FM(TAUD0O13)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
 #define IP1SR4_23_20	F_(0, 0)		F_(0, 0)		F_(0, 0)		F_(0, 0)		F_(0, 0)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP1SR4_27_24	FM(GP4_14)		FM(ERRORIN1_N)		F_(0, 0)		FM(TAUD0I15)		FM(TAUD0O15)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP1SR4_31_28	FM(GP4_15)		FM(MSPI1CSS3)		F_(0, 0)		FM(TAUD1I1)		FM(TAUD1O1)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP1SR4_27_24	F_(0, 0)		FM(ERRORIN1_N)		F_(0, 0)		FM(TAUD0I15)		FM(TAUD0O15)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP1SR4_31_28	F_(0, 0)		FM(MSPI1CSS3)		F_(0, 0)		FM(TAUD1I1)		FM(TAUD1O1)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
 /* IP2SR4 */		/* 0 */			/* 1 */			/* 2 */			/* 3 */			/* 4 */		/* 5 */			/* 6 - F */
-#define IP2SR4_3_0	FM(GP4_16)		F_(0, 0)		F_(0, 0)		FM(TAUD1I0)		FM(TAUD1O0)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP2SR4_7_4	FM(GP4_17)		FM(MSPI1CSS5)		F_(0, 0)		FM(TAUD1I3)		FM(TAUD1O3)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP2SR4_11_8	FM(GP4_18)		FM(MSPI1CSS4)		F_(0, 0)		FM(TAUD1I2)		FM(TAUD1O2)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
-#define IP2SR4_15_12	FM(GP4_19)		FM(MSPI1CSS6)		F_(0, 0)		FM(TAUD1I4)		FM(TAUD1O4)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP2SR4_3_0	F_(0, 0)		F_(0, 0)		F_(0, 0)		FM(TAUD1I0)		FM(TAUD1O0)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP2SR4_7_4	F_(0, 0)		FM(MSPI1CSS5)		F_(0, 0)		FM(TAUD1I3)		FM(TAUD1O3)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP2SR4_11_8	F_(0, 0)		FM(MSPI1CSS4)		F_(0, 0)		FM(TAUD1I2)		FM(TAUD1O2)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
+#define IP2SR4_15_12	F_(0, 0)		FM(MSPI1CSS6)		F_(0, 0)		FM(TAUD1I4)		FM(TAUD1O4)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
 #define IP2SR4_19_16	FM(MSPI0SC)		FM(MSPI1CSS7)		F_(0, 0)		FM(TAUD1I5)		FM(TAUD1O5)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
 #define IP2SR4_23_20	FM(MSPI0SI)		F_(0, 0)		F_(0, 0)		FM(TAUD1I7)		FM(TAUD1O7)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
 #define IP2SR4_27_24	FM(MSPI0SO_MSPI0DCS)	F_(0, 0)		F_(0, 0)		FM(TAUD1I6)		FM(TAUD1O6)	F_(0, 0)		F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0) F_(0, 0)
@@ -712,25 +712,21 @@ static const u16 pinmux_data[] = {
 
 	/* IP0SR1 */
 	/* GP1_00 = SCL0 */
-	PINMUX_IPSR_MSEL(IP0SR1_3_0,	GP1_00,	SEL_I2C0_0),
 	PINMUX_IPSR_MSEL(IP0SR1_3_0,	TCLK1,	SEL_I2C0_0),
 	PINMUX_IPSR_MSEL(IP0SR1_3_0,	HSCK2,	SEL_I2C0_0),
 	PINMUX_IPSR_PHYS(IP0SR1_3_0,	SCL0,	SEL_I2C0_3),
 
 	/* GP1_01 = SDA0 */
-	PINMUX_IPSR_MSEL(IP0SR1_7_4,	GP1_01,	SEL_I2C0_0),
 	PINMUX_IPSR_MSEL(IP0SR1_7_4,	TCLK4,	SEL_I2C0_0),
 	PINMUX_IPSR_MSEL(IP0SR1_7_4,	HRX2,	SEL_I2C0_0),
 	PINMUX_IPSR_PHYS(IP0SR1_7_4,	SDA0,	SEL_I2C0_3),
 
 	/* GP1_02 = SCL1 */
-	PINMUX_IPSR_MSEL(IP0SR1_11_8,	GP1_02,		SEL_I2C1_0),
 	PINMUX_IPSR_MSEL(IP0SR1_11_8,	HTX2,		SEL_I2C1_0),
 	PINMUX_IPSR_MSEL(IP0SR1_11_8,	MSIOF2_SS1,	SEL_I2C1_0),
 	PINMUX_IPSR_PHYS(IP0SR1_11_8,	SCL1,		SEL_I2C1_3),
 
 	/* GP1_03 = SDA1 */
-	PINMUX_IPSR_MSEL(IP0SR1_15_12,	GP1_03,		SEL_I2C1_0),
 	PINMUX_IPSR_MSEL(IP0SR1_15_12,	TCLK2,		SEL_I2C1_0),
 	PINMUX_IPSR_MSEL(IP0SR1_15_12,	HCTS2_N,	SEL_I2C1_0),
 	PINMUX_IPSR_MSEL(IP0SR1_15_12,	MSIOF2_SS2,	SEL_I2C1_0),
@@ -738,7 +734,6 @@ static const u16 pinmux_data[] = {
 	PINMUX_IPSR_PHYS(IP0SR1_15_12,	SDA1,		SEL_I2C1_3),
 
 	/* GP1_04 = SCL2 */
-	PINMUX_IPSR_MSEL(IP0SR1_19_16,	GP1_04,		SEL_I2C2_0),
 	PINMUX_IPSR_MSEL(IP0SR1_19_16,	TCLK3,		SEL_I2C2_0),
 	PINMUX_IPSR_MSEL(IP0SR1_19_16,	HRTS2_N,	SEL_I2C2_0),
 	PINMUX_IPSR_MSEL(IP0SR1_19_16,	MSIOF2_SYNC,	SEL_I2C2_0),
@@ -746,19 +741,16 @@ static const u16 pinmux_data[] = {
 	PINMUX_IPSR_PHYS(IP0SR1_19_16,	SCL2,		SEL_I2C2_3),
 
 	/* GP1_05 = SDA2 */
-	PINMUX_IPSR_MSEL(IP0SR1_23_20,	GP1_05,		SEL_I2C2_0),
 	PINMUX_IPSR_MSEL(IP0SR1_23_20,	MSIOF2_SCK,	SEL_I2C2_0),
 	PINMUX_IPSR_MSEL(IP0SR1_23_20,	SCK4,		SEL_I2C2_0),
 	PINMUX_IPSR_PHYS(IP0SR1_23_20,	SDA2,		SEL_I2C2_3),
 
 	/* GP1_06 = SCL3 */
-	PINMUX_IPSR_MSEL(IP0SR1_27_24,	GP1_06,		SEL_I2C3_0),
 	PINMUX_IPSR_MSEL(IP0SR1_27_24,	MSIOF2_RXD,	SEL_I2C3_0),
 	PINMUX_IPSR_MSEL(IP0SR1_27_24,	RX4,		SEL_I2C3_0),
 	PINMUX_IPSR_PHYS(IP0SR1_27_24,	SCL3,		SEL_I2C3_3),
 
 	/* GP1_07 = SDA3 */
-	PINMUX_IPSR_MSEL(IP0SR1_31_28,	GP1_07,		SEL_I2C3_0),
 	PINMUX_IPSR_MSEL(IP0SR1_31_28,	MSIOF2_TXD,	SEL_I2C3_0),
 	PINMUX_IPSR_MSEL(IP0SR1_31_28,	TX4,		SEL_I2C3_0),
 	PINMUX_IPSR_PHYS(IP0SR1_31_28,	SDA3,		SEL_I2C3_3),
@@ -780,107 +772,88 @@ static const u16 pinmux_data[] = {
 	PINMUX_IPSR_PHYS_NOFN(SDA5,	SEL_I2C5_3),
 
 	/* IP0SR4 */
-	PINMUX_IPSR_GPSR(IP0SR4_3_0,	GP4_00),
 	PINMUX_IPSR_GPSR(IP0SR4_3_0,	MSPI4SC),
 	PINMUX_IPSR_GPSR(IP0SR4_3_0,	TAUD0I2),
 	PINMUX_IPSR_GPSR(IP0SR4_3_0,	TAUD0O2),
 
-	PINMUX_IPSR_GPSR(IP0SR4_7_4,	GP4_01),
 	PINMUX_IPSR_GPSR(IP0SR4_7_4,	MSPI4SI),
 	PINMUX_IPSR_GPSR(IP0SR4_7_4,	TAUD0I4),
 	PINMUX_IPSR_GPSR(IP0SR4_7_4,	TAUD0O4),
 
-	PINMUX_IPSR_GPSR(IP0SR4_11_8,	GP4_02),
 	PINMUX_IPSR_GPSR(IP0SR4_11_8,	MSPI4SO_MSPI4DCS),
 	PINMUX_IPSR_GPSR(IP0SR4_11_8,	TAUD0I3),
 	PINMUX_IPSR_GPSR(IP0SR4_11_8,	TAUD0O3),
 
-	PINMUX_IPSR_GPSR(IP0SR4_15_12,	GP4_03),
 	PINMUX_IPSR_GPSR(IP0SR4_15_12,	MSPI4CSS1),
 	PINMUX_IPSR_GPSR(IP0SR4_15_12,	TAUD0I6),
 	PINMUX_IPSR_GPSR(IP0SR4_15_12,	TAUD0O6),
 	PINMUX_IPSR_GPSR(IP0SR4_15_12,	MSPI5SO_MSPI5DCS),
 
-	PINMUX_IPSR_GPSR(IP0SR4_19_16,	GP4_04),
 	PINMUX_IPSR_GPSR(IP0SR4_19_16,	MSPI4CSS0),
 	PINMUX_IPSR_GPSR(IP0SR4_19_16,	MSPI4SSI_N),
 	PINMUX_IPSR_GPSR(IP0SR4_19_16,	TAUD0I5),
 	PINMUX_IPSR_GPSR(IP0SR4_19_16,	TAUD0O5),
 	PINMUX_IPSR_GPSR(IP0SR4_19_16,	MSPI5SC),
 
-	PINMUX_IPSR_GPSR(IP0SR4_23_20,	GP4_05),
 	PINMUX_IPSR_GPSR(IP0SR4_23_20,	MSPI4CSS3),
 	PINMUX_IPSR_GPSR(IP0SR4_23_20,	TAUD0I8),
 	PINMUX_IPSR_GPSR(IP0SR4_23_20,	TAUD0O8),
 	PINMUX_IPSR_GPSR(IP0SR4_23_20,	MSPI5SSI_N),
 
-	PINMUX_IPSR_GPSR(IP0SR4_27_24,	GP4_06),
 	PINMUX_IPSR_GPSR(IP0SR4_27_24,	MSPI4CSS2),
 	PINMUX_IPSR_GPSR(IP0SR4_27_24,	TAUD0I7),
 	PINMUX_IPSR_GPSR(IP0SR4_27_24,	TAUD0O7),
 	PINMUX_IPSR_GPSR(IP0SR4_27_24,	MSPI5SI),
 
-	PINMUX_IPSR_GPSR(IP0SR4_31_28,	GP4_07),
 	PINMUX_IPSR_GPSR(IP0SR4_31_28,	MSPI4CSS5),
 	PINMUX_IPSR_GPSR(IP0SR4_31_28,	TAUD0I10),
 	PINMUX_IPSR_GPSR(IP0SR4_31_28,	TAUD0O10),
 	PINMUX_IPSR_GPSR(IP0SR4_31_28,	MSPI5CSS1),
 
 	/* IP1SR4 */
-	PINMUX_IPSR_GPSR(IP1SR4_3_0,	GP4_08),
 	PINMUX_IPSR_GPSR(IP1SR4_3_0,	MSPI4CSS4),
 	PINMUX_IPSR_GPSR(IP1SR4_3_0,	TAUD0I9),
 	PINMUX_IPSR_GPSR(IP1SR4_3_0,	TAUD0O9),
 	PINMUX_IPSR_GPSR(IP1SR4_3_0,	MSPI5CSS0),
 
-	PINMUX_IPSR_GPSR(IP1SR4_7_4,	GP4_09),
 	PINMUX_IPSR_GPSR(IP1SR4_7_4,	MSPI4CSS7),
 	PINMUX_IPSR_GPSR(IP1SR4_7_4,	TAUD0I12),
 	PINMUX_IPSR_GPSR(IP1SR4_7_4,	TAUD0O12),
 	PINMUX_IPSR_GPSR(IP1SR4_7_4,	MSPI5CSS3),
 
-	PINMUX_IPSR_GPSR(IP1SR4_11_8,	GP4_10),
 	PINMUX_IPSR_GPSR(IP1SR4_11_8,	MSPI4CSS6),
 	PINMUX_IPSR_GPSR(IP1SR4_11_8,	TAUD0I11),
 	PINMUX_IPSR_GPSR(IP1SR4_11_8,	TAUD0O11),
 	PINMUX_IPSR_GPSR(IP1SR4_11_8,	MSPI5CSS2),
 
-	PINMUX_IPSR_GPSR(IP1SR4_15_12,	GP4_11),
 	PINMUX_IPSR_GPSR(IP1SR4_15_12,	ERRORIN0_N),
 	PINMUX_IPSR_GPSR(IP1SR4_15_12,	TAUD0I14),
 	PINMUX_IPSR_GPSR(IP1SR4_15_12,	TAUD0O14),
 
-	PINMUX_IPSR_GPSR(IP1SR4_19_16,	GP4_12),
 	PINMUX_IPSR_GPSR(IP1SR4_19_16,	ERROROUT_C_N),
 	PINMUX_IPSR_GPSR(IP1SR4_19_16,	TAUD0I13),
 	PINMUX_IPSR_GPSR(IP1SR4_19_16,	TAUD0O13),
 
-	PINMUX_IPSR_GPSR(IP1SR4_27_24,	GP4_14),
 	PINMUX_IPSR_GPSR(IP1SR4_27_24,	ERRORIN1_N),
 	PINMUX_IPSR_GPSR(IP1SR4_27_24,	TAUD0I15),
 	PINMUX_IPSR_GPSR(IP1SR4_27_24,	TAUD0O15),
 
-	PINMUX_IPSR_GPSR(IP1SR4_31_28,	GP4_15),
 	PINMUX_IPSR_GPSR(IP1SR4_31_28,	MSPI1CSS3),
 	PINMUX_IPSR_GPSR(IP1SR4_31_28,	TAUD1I1),
 	PINMUX_IPSR_GPSR(IP1SR4_31_28,	TAUD1O1),
 
 	/* IP2SR4 */
-	PINMUX_IPSR_GPSR(IP2SR4_3_0,	GP4_16),
 	PINMUX_IPSR_GPSR(IP2SR4_3_0,	TAUD1I0),
 	PINMUX_IPSR_GPSR(IP2SR4_3_0,	TAUD1O0),
 
-	PINMUX_IPSR_GPSR(IP2SR4_7_4,	GP4_17),
 	PINMUX_IPSR_GPSR(IP2SR4_7_4,	MSPI1CSS5),
 	PINMUX_IPSR_GPSR(IP2SR4_7_4,	TAUD1I3),
 	PINMUX_IPSR_GPSR(IP2SR4_7_4,	TAUD1O3),
 
-	PINMUX_IPSR_GPSR(IP2SR4_11_8,	GP4_18),
 	PINMUX_IPSR_GPSR(IP2SR4_11_8,	MSPI1CSS4),
 	PINMUX_IPSR_GPSR(IP2SR4_11_8,	TAUD1I2),
 	PINMUX_IPSR_GPSR(IP2SR4_11_8,	TAUD1O2),
 
-	PINMUX_IPSR_GPSR(IP2SR4_15_12,	GP4_19),
 	PINMUX_IPSR_GPSR(IP2SR4_15_12,	MSPI1CSS6),
 	PINMUX_IPSR_GPSR(IP2SR4_15_12,	TAUD1I4),
 	PINMUX_IPSR_GPSR(IP2SR4_15_12,	TAUD1O4),
-- 
2.34.1

